### ISSUE TITLE
Parseable output including severity and pep8 not working

### ISSUE THREAD

**Issue by JiriSko** [ansible/ansible-lint#1054]
##### Summary
Parseable-severity mode together with parseable output in the format of pep8 doesn't show severity in output.

##### Issue Type

- Bug Report

##### Ansible and Ansible Lint details
```console
bash-4.2# ansible --version
ansible 2.9.13
  config file = None
  configured module search path = ['/root/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /var/task/python/lib/python3.8/site-packages/ansible
  executable location = /var/task/python/bin/ansible
  python version = 3.8.5 (default, Sep 14 2020, 06:51:44) [GCC 7.3.1 20180712 (Red Hat 7.3.1-6)]
bash-4.2# ansible-lint --version
ansible-lint 4.3.4
```

- ansible installation method: pip
- ansible-lint installation method: pip

##### OS / ENVIRONMENT
Custom docker image based on `amazonlinux:2.0.20190508` with installed python 3.8.5

##### STEPS TO REPRODUCE
```console
ansible-lint -p --parseable-severity roles/test.yml
```

##### Desired Behaviour
```
roles/test.yml:5: [E301] [HIGH] Commands should not change things if nothing needs doing
roles/test.yml:5: [E303] [HIGH] service used in place of service module
roles/test.yml:8: [E301] [HIGH] Commands should not change things if nothing needs doing
roles/test.yml:8: [E303] [HIGH] service used in place of service module
roles/test.yml:13: [E301] [HIGH] Commands should not change things if nothing needs doing
roles/test.yml:13: [E303] [HIGH] service used in place of service module
roles/test.yml:13: [E502] [MEDIUM] All tasks should be named
```

This behavior is captured from ansible-lint v4.2.0.

##### Actual Behaviour
```
roles/test.yml:5: [E301] Commands should not change things if nothing needs doing
roles/test.yml:5: [E303] service used in place of service module
roles/test.yml:8: [E301] Commands should not change things if nothing needs doing
roles/test.yml:8: [E303] service used in place of service module
roles/test.yml:13: [E301] Commands should not change things if nothing needs doing
roles/test.yml:13: [E303] service used in place of service module
roles/test.yml:13: [E502] All tasks should be named
```

`roles/test.yml` contents:
```yml
---
- hosts: bobbins

  tasks:
  - name: a bad play
    action: command service blah restart

  - name: Start Nginx
    command: service nginx start
    tags:
      - nginx

  - command: service nginx start
    tags:
      - nginx
```

**Comment by ssbarnea** [comment]
What was your expectation around this considering the pep8 format does not allow display of severity?

**Comment by JiriSko** [comment]
Oh, I didn't know that because it works in previous version. I'm looking for a machine-readable format including severity. Is any alternative?

**Comment by ssbarnea** [comment]
Your best bet is to use github annotation format, as that one seems to be extensible, and we already support it. See https://ansible-lint.readthedocs.io/en/latest/usage.html#ci-cd

I am not sure if we output severity with it but if not you can easily make a patch to include it.

**Comment by JiriSko** [comment]
It look that severity isn't included in the output (tested).

https://github.com/ansible/ansible-lint/blob/master/lib/ansiblelint/formatters/__init__.py#L127 , patch should be simply, like you said.

**Comment by JiriSko** [comment]
I've created PR #1055. Is that what you thought?
